docs/02-guides/display-a-map.mdx

```diff
@@ -0,0 +1,104 @@
+| ------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| location      | Location for the marker                | [`LocationInputShape`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/location.js#L65-74) |
+| icon          | Allows you to override the marker icon | [`object`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/index.js#L25-31)                |
+| isPopupOpened | Controlled method for marker popup     | `boolean`                                                                                                                                                                                       |
+| onClick       | Event handler on marker click          | `(event) => void`                                                                                                                                                                               |
+
+### Install a map extension
+
+Follow the installation instruction of the map extension you want to use:
+
+- [Open Street Map (OSM) with Leaflet](/docs/extensions/leaflet#installation)
+- [Google Maps](/docs/extensions/google-maps#installation)
+
+After this, you should be able to use the Map component. See
+[Map.story.js](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/modules/map-googlemap/web/theme/components/organisms/Map/Map.story.js)
+for details.
+
+:::note
+
+_Please keep in mind, that the Map component should be used in an element with a
+fixed height._
+
+:::
+
+### Display a map
+
+Using
+[`useExtensionComponentMap`](/docs/3.x/api-reference/front-commerce-core/react#useextensioncomponentmap),
+you can retrieve the registered map components and use it in your application:
+
+```tsx title="theme/myMapComponent.tsx"
+import { useCallback } from "react";
+import { useExtensionComponentMap } from "@front-commerce/core/react";
+
+export default function MyMapComponent() {
+  const MapComponents = useExtensionComponentMap("maps");
+  const locations = [
+    {
+      lat: 48.8566,
+      lng: 2.3522,
+    },
+    {
+      lat: 48.8536,
+      lng: 2.3523,
+    },
+  ];
+
+  return (
+    <div style={{ height: "500px" }}>
+      <MapComponents.Map
+        locations={locations}
+        getMarker={useCallback((location) => {
+          const LocationMarkerComponent = location.getMarkerComponent();
+          return <MapComponents.Marker key={location.id} location={location} />;
+        }, [])}
+      />
+    </div>
+  );
+}
+```
```

Comment:
I would also include a section of how to override the map with a custom component in the extension config.

Reply:
Added such a section, that's indeed nice.